diff --git a/frontend/package-lock.json b/frontend/package-lock.json
index d2ff583..a9e42ef 100644
--- a/frontend/package-lock.json
+++ b/frontend/package-lock.json
@@ -17,7 +17,6 @@
         "react": "^19.1.0",
         "react-dom": "^19.1.0",
         "react-router": "^7.6.0",
-        "react-scroll": "^1.9.3",
         "tailwindcss": "^4.1.7",
         "three": "^0.177.0"
       },
@@ -2575,12 +2574,6 @@
         "jiti": "lib/jiti-cli.mjs"
       }
     },
-    "node_modules/js-tokens": {
-      "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/js-tokens/-/js-tokens-4.0.0.tgz",
-      "integrity": "sha512-RdJUflcE3cUzKiMqQgsCu06FPu9UdIJO0beYbPhHN4k6apgJtifcoCtT9bcxOpYBtpD2kCM6Sbzg4CausW/PKQ==",
-      "license": "MIT"
-    },
     "node_modules/js-yaml": {
       "version": "4.1.0",
       "resolved": "https://registry.npmjs.org/js-yaml/-/js-yaml-4.1.0.tgz",
@@ -2899,24 +2892,6 @@
       "dev": true,
       "license": "MIT"
     },
-    "node_modules/lodash.throttle": {
-      "version": "4.1.1",
-      "resolved": "https://registry.npmjs.org/lodash.throttle/-/lodash.throttle-4.1.1.tgz",
-      "integrity": "sha512-wIkUCfVKpVsWo3JSZlc+8MB5it+2AN5W8J7YVMST30UrvcQNZ1Okbj+rbVniijTWE6FGYy4XJq/rHkas8qJMLQ==",
-      "license": "MIT"
-    },
-    "node_modules/loose-envify": {
-      "version": "1.4.0",
-      "resolved": "https://registry.npmjs.org/loose-envify/-/loose-envify-1.4.0.tgz",
-      "integrity": "sha512-lyuxPGr/Wfhrlem2CL/UcnUc1zcqKAImBDzukY7Y5F/yQiNdko6+fRLevlw1HgMySw7f611UIY408EtxRSoK3Q==",
-      "license": "MIT",
-      "dependencies": {
-        "js-tokens": "^3.0.0 || ^4.0.0"
-      },
-      "bin": {
-        "loose-envify": "cli.js"
-      }
-    },
     "node_modules/maath": {
       "version": "0.10.8",
       "resolved": "https://registry.npmjs.org/maath/-/maath-0.10.8.tgz",
@@ -3032,15 +3007,6 @@
       "dev": true,
       "license": "MIT"
     },
-    "node_modules/object-assign": {
-      "version": "4.1.1",
-      "resolved": "https://registry.npmjs.org/object-assign/-/object-assign-4.1.1.tgz",
-      "integrity": "sha512-rJgTQnkUnH1sFw8yT6VSU3zD3sWmu6sZhIseY8VX+GRu3P6F7Fu+JNDoXfklElbLJSnc3FUQHVe4cU5hj+BcUg==",
-      "license": "MIT",
-      "engines": {
-        "node": ">=0.10.0"
-      }
-    },
     "node_modules/optionator": {
       "version": "0.9.4",
       "resolved": "https://registry.npmjs.org/optionator/-/optionator-0.9.4.tgz",
@@ -3195,17 +3161,6 @@
         "lie": "^3.0.2"
       }
     },
-    "node_modules/prop-types": {
-      "version": "15.8.1",
-      "resolved": "https://registry.npmjs.org/prop-types/-/prop-types-15.8.1.tgz",
-      "integrity": "sha512-oj87CgZICdulUohogVAR7AjlC0327U4el4L6eAvOqCeudMDVU0NThNaV+b9Df4dXgSP1gXMTnPdhfe/2qDH5cg==",
-      "license": "MIT",
-      "dependencies": {
-        "loose-envify": "^1.4.0",
-        "object-assign": "^4.1.1",
-        "react-is": "^16.13.1"
-      }
-    },
     "node_modules/punycode": {
       "version": "2.3.1",
       "resolved": "https://registry.npmjs.org/punycode/-/punycode-2.3.1.tgz",
@@ -3237,12 +3192,6 @@
         "react": "^19.1.0"
       }
     },
-    "node_modules/react-is": {
-      "version": "16.13.1",
-      "resolved": "https://registry.npmjs.org/react-is/-/react-is-16.13.1.tgz",
-      "integrity": "sha512-24e6ynE2H+OKt4kqsOvNd8kBpV65zoxbA4BVsEOB3ARVWQki/DHzaUoC5KuON/BiccDaCCTZBuOcfZs70kR8bQ==",
-      "license": "MIT"
-    },
     "node_modules/react-reconciler": {
       "version": "0.31.0",
       "resolved": "https://registry.npmjs.org/react-reconciler/-/react-reconciler-0.31.0.tgz",
@@ -3286,20 +3235,6 @@
         }
       }
     },
-    "node_modules/react-scroll": {
-      "version": "1.9.3",
-      "resolved": "https://registry.npmjs.org/react-scroll/-/react-scroll-1.9.3.tgz",
-      "integrity": "sha512-xv7FXqF3k63aSLNu4/NjFvRNI0ge7DmmmsbeGarP7LZVAlJMSjUuW3dTtLxp1Afijyv0lS2qwC0GiFHvx1KBHQ==",
-      "license": "MIT",
-      "dependencies": {
-        "lodash.throttle": "^4.1.1",
-        "prop-types": "^15.7.2"
-      },
-      "peerDependencies": {
-        "react": "^15.5.4 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0",
-        "react-dom": "^15.5.4 || ^16.0.0 || ^17.0.0 || ^18.0.0 || ^19.0.0"
-      }
-    },
     "node_modules/react-use-measure": {
       "version": "2.1.7",
       "resolved": "https://registry.npmjs.org/react-use-measure/-/react-use-measure-2.1.7.tgz",
diff --git a/frontend/package.json b/frontend/package.json
index 92ce6ae..632024a 100644
--- a/frontend/package.json
+++ b/frontend/package.json
@@ -19,7 +19,6 @@
     "react": "^19.1.0",
     "react-dom": "^19.1.0",
     "react-router": "^7.6.0",
-    "react-scroll": "^1.9.3",
     "tailwindcss": "^4.1.7",
     "three": "^0.177.0"
   },
diff --git a/frontend/src/components/layout/home_route/Home.jsx b/frontend/src/components/layout/home_route/Home.jsx
index a862025..e8d2037 100755
--- a/frontend/src/components/layout/home_route/Home.jsx
+++ b/frontend/src/components/layout/home_route/Home.jsx
@@ -1,5 +1,4 @@
 import { Suspense, useEffect, useState } from "react";
-import { Element } from "react-scroll";
 import LoadingScreen from "../../../components/ui_components/LoadingScreen";
 import NavBar from "../../ui_components/home_route/navbar/NavBar";
 import Footer from "../../ui_components/home_route/footer/Footer";
@@ -32,49 +31,53 @@ const Home = () => {
             <div className="fixed inset-0 w-full h-full -z-10 bg-base-300/20 landscape:backdrop-blur-sm backdrop-blur-xs dark:landscape:backdrop-blur-md"></div>
             <div className="h-[64px]"></div>
             <Suspense fallback={null}>
-              <Element
-                name="landing"
+              <section
+                id="landing"
                 className="active bg-gradient-to-r from-base-200 to-base-200/0 via-base-200/20"
               >
                 <div className="bg-gradient-to-b from-base-300 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
                   <Landing />
                 </div>
-              </Element>
-              <div className="">
-                <Element
-                  name="about"
-                  className="bg-gradient-to-b landscape:bg-gradient-to-r from-base-200/0 to-base-200 via-base-200/50 landscape:via-base-200/80"
-                >
-                  <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
-                    <About />
-                  </div>
-                </Element>
-                <Element name="services" className="">
-                  <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
-                    <Services />
-                  </div>
-                </Element>
-                <Element name="gallery" className="">
-                  <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
-                    <Gallery />
-                  </div>
-                </Element>
-                <Element name="contact" className="">
-                  <div
-                    id="contact"
-                    className="bg-gradient-to-b from-base-200 to-base-300 via-base-200/0 landscape:h-[calc(100dvh_-_64px)] flex items-center justify-center"
-                  >
-                    <ContactUs />
-                  </div>
-                </Element>
+              </section>
+              <section
+                id="about"
+                className="bg-gradient-to-b landscape:bg-gradient-to-r from-base-200/0 to-base-200 via-base-200/50 landscape:via-base-200/80"
+              >
+                <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
+                  <About />
+                </div>
+              </section>
+              <div className="bg-base-200 landscape:hidden">
+                <div className="w-[90%] h-0.5 ml-[5%] bg-cyber_teal"></div>
+              </div>
+              <section id="services" className="">
+                <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0  min-h-[calc(100dvh_-_64px)] flex items-center justify-center">
+                  <Services />
+                </div>
+              </section>
+              <div className="bg-base-200 landscape:hidden">
+                <div className="w-[90%] h-0.5 ml-[5%] bg-cyber_teal"></div>
               </div>
+              <section id="gallery" className="">
+                <div className="bg-gradient-to-b from-base-200 to-base-200 via-base-200/0 h-[calc(100dvh_-_64px)] flex items-center justify-center">
+                  <Gallery />
+                </div>
+              </section>
+              <section id="contact" className="">
+                <div
+                  id="contact"
+                  className="bg-gradient-to-b from-base-200 to-base-300 via-base-200/0 landscape:h-[calc(100dvh_-_64px)] flex items-center justify-center"
+                >
+                  <ContactUs />
+                </div>
+              </section>
             </Suspense>
           </div>
         }
       />
-      <Element name="footer" className="">
+      <section id="footer" className="">
         <Footer />
-      </Element>
+      </section>
     </>
   );
 };
diff --git a/frontend/src/components/layout/home_route/sections/About.jsx b/frontend/src/components/layout/home_route/sections/About.jsx
index bac5641..b818940 100644
--- a/frontend/src/components/layout/home_route/sections/About.jsx
+++ b/frontend/src/components/layout/home_route/sections/About.jsx
@@ -3,10 +3,13 @@ import CamerasOnPoleIcon from "../../../ui_components/icons/CamerasOnPoleIcon";
 const About = () => {
   return (
     <>
-      <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] flex landscape:flex-row flex-col gap-4 landscape:gap-0">
+      <div
+        id="about"
+        className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] flex landscape:flex-row flex-col gap-4 landscape:gap-0"
+      >
         <div className="flex-1 flex justify-center items-center landscape:px-8 px-4">
           <div className="max-h-full w-full grow flex items-center justify-center">
-            <p className="overflow-y-scroll grow landscape:max-h-[calc(100dvh_-_150px)] max-h-[calc((100dvh_-_150px)_/_2)] leading-relaxed">
+            <p className="overflow-y-scroll scroll-smooth grow landscape:max-h-[calc(100dvh_-_150px)] max-h-[calc((100dvh_-_150px)_/_2)] leading-relaxed">
               <span className="text-cyber_teal font-semibold text-shadow-sm text-shadow-cyber_teal_tint">
                 Surveillance SA{" "}
               </span>
diff --git a/frontend/src/components/layout/home_route/sections/ContactUs.jsx b/frontend/src/components/layout/home_route/sections/ContactUs.jsx
index 0a3766f..efb9af3 100644
--- a/frontend/src/components/layout/home_route/sections/ContactUs.jsx
+++ b/frontend/src/components/layout/home_route/sections/ContactUs.jsx
@@ -4,7 +4,10 @@ import Map from "../../../ui_components/home_route/contact_us/ContactMap";
 const ContactUs = () => {
   return (
     <>
-      <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-auto flex landscape:flex-row portrait:flex-col portrait:py-8 gap-16">
+      <div
+        id="contact"
+        className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-auto flex landscape:flex-row portrait:flex-col portrait:py-8 gap-16"
+      >
         <div className="flex-1 bg-base-200">
           <ContactForm />
         </div>
diff --git a/frontend/src/components/layout/home_route/sections/Gallery.jsx b/frontend/src/components/layout/home_route/sections/Gallery.jsx
index 5b66c77..476e94d 100644
--- a/frontend/src/components/layout/home_route/sections/Gallery.jsx
+++ b/frontend/src/components/layout/home_route/sections/Gallery.jsx
@@ -31,7 +31,10 @@ const Gallery = () => {
   return (
     <>
       {isLandscape ? (
-        <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] rounded-sm overflow-hidden carousel carousel-center bg-neutral space-x-4">
+        <div
+          id="gallery"
+          className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] rounded-sm overflow-hidden carousel carousel-center bg-neutral space-x-4"
+        >
           <div
             id="slide1"
             className="relative carousel-item w-full flex items-center justify-center"
@@ -264,7 +267,10 @@ const Gallery = () => {
           </div>
         </div>
       ) : (
-        <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] carousel carousel-center bg-neutral space-x-4">
+        <div
+          id="gallery"
+          className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] carousel carousel-center bg-neutral space-x-4"
+        >
           <div
             id="slide1"
             className="relative carousel-item w-full flex items-center justify-center"
diff --git a/frontend/src/components/layout/home_route/sections/Landing.jsx b/frontend/src/components/layout/home_route/sections/Landing.jsx
index c693d0c..09a58c0 100644
--- a/frontend/src/components/layout/home_route/sections/Landing.jsx
+++ b/frontend/src/components/layout/home_route/sections/Landing.jsx
@@ -1,4 +1,3 @@
-import { Link } from "react-scroll";
 import LandingMap from "../../../ui_components/home_route/landing/LandingMap";
 import Globe from "../../../ui_components/home_route/landing/globe/Globe";
 import useIsLandscape from "../../../../hooks/useIsLandscape";
@@ -7,7 +6,10 @@ const Landing = () => {
   const isLandscape = useIsLandscape();
   return (
     <>
-      <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_64px)] flex flex-col-reverse landscape:flex-row">
+      <div
+        id="landing"
+        className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_64px)] flex flex-col-reverse landscape:flex-row"
+      >
         <div className="flex-1">
           <div className="h-[calc((100dvh_-_64px)_/_2)] landscape:h-[calc(100dvh_-_64px)] flex flex-col justify-center items-center landscape:items-start gap-4">
             <h1 className="font-bold text-4xl md:text-5xl 2xl:text-7xl leading-tight text-center landscape:text-left">
@@ -28,19 +30,11 @@ const Landing = () => {
               <br />
               Proactive security and intelligent monitoring in real time.
             </p>
-            <Link
-              to="contact"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass=""
-              className=""
-            >
+            <a href="#contact" className="">
               <button className="btn btn-primary text-base-content">
                 Get Started
               </button>
-            </Link>
+            </a>
           </div>
         </div>
         <div className="flex flex-1 items-center justify-center">
diff --git a/frontend/src/components/layout/home_route/sections/Services.jsx b/frontend/src/components/layout/home_route/sections/Services.jsx
index 6353b9c..1f5290d 100644
--- a/frontend/src/components/layout/home_route/sections/Services.jsx
+++ b/frontend/src/components/layout/home_route/sections/Services.jsx
@@ -3,141 +3,59 @@ import CamerasOnPoleIcon from "../../../ui_components/icons/CamerasOnPoleIcon";
 const Services = () => {
   return (
     <>
-      {/* name of each tab group should be unique */}
-      <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] flex items-center justify-center">
-        <div className="tabs tabs-lift w-full">
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" defaultChecked />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 1
-          </label>
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-contain bg-repeat-round bg-left-top bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)] flex flex-col items-center gap-8">
-              <div className="flex flex-col justify-center items-center gap-4">
-                <h2 className="text-cyber_teal font-bold text-3xl text-center">
-                  Surveillance and System Health Monitoring
-                </h2>
-                <p className="text-center text-sm max-w-[70%]">
-                  We provide uninterrupted protection and peace of mind.
-                  Ensuring that even if the camera is stolen, damaged or loses
-                  connection, your footage is always safe and your system is
-                  always under watch.
-                </p>
+      <div className="flex flex-col">
+        <div
+          id="service_1"
+          className="w-full h-[calc(100dvh_-_64px)] bg-base-200/20 backdrop-blur-xs landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat flex justify-center"
+        >
+          <div className="w-[80%] flex flex-col justify-evenly">
+            <div className="flex flex-col items-center landscape:items-start justify-center gap-4">
+              <h2 className="text-cyber_teal landscape:text-base-content landscape:dark:text-cyber_teal font-bold text-4xl md:text-5xl text-center landscape:text-left leading-tight">
+                <span className="text-base-content landscape:text-cyber_teal landscape:dark:text-base-content">
+                  [{" "}
+                </span>
+                Surveillance and System Health Monitoring
+                <span className="text-base-content landscape:text-cyber_teal landscape:dark:text-base-content">
+                  {" "}
+                  ]
+                </span>
+              </h2>
+              <p className="text-md landscape:max-w-[70%] text-center landscape:text-left leading-tight tracking-wide">
+                We provide uninterrupted protection and peace of mind. Ensuring
+                that even if the camera is stolen, damaged or loses connection,
+                your footage is always safe and your system is always under
+                watch.
+              </p>
+            </div>
+            <div className="w-full flex flex-col gap-4">
+              <div className="flex w-full flex-row justify-start items-center gap-4">
+                <CamerasOnPoleIcon className="w-[48px] h-[48px]" />
+                <h3 className="text-xl font-semibold">
+                  REDUNDANT FOOTAGE BACKUP
+                </h3>
               </div>
-              <div className="w-full flex flex-col gap-4">
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    REDUNDANT FOOTAGE BACKUP
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    LIVE SYSTEM HEALTH MONITORING
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    LESS DOWNTIME GUARANTEE
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">TRUE PEACE OF MIND</h3>
-                </div>
+              <div className="flex w-full flex-row justify-start items-center gap-4">
+                <CamerasOnPoleIcon className="w-[48px] h-[48px]" />
+                <h3 className="text-xl font-semibold">
+                  LIVE SYSTEM HEALTH MONITORING
+                </h3>
+              </div>
+              <div className="flex w-full flex-row justify-start items-center gap-4">
+                <CamerasOnPoleIcon className="w-[48px] h-[48px]" />
+                <h3 className="text-xl font-semibold">
+                  LESS DOWNTIME GUARANTEE
+                </h3>
+              </div>
+              <div className="flex w-full flex-row justify-start items-center gap-4">
+                <CamerasOnPoleIcon className="w-[48px] h-[48px]" />
+                <h3 className="text-xl font-semibold">TRUE PEACE OF MIND</h3>
               </div>
             </div>
           </div>
+        </div>
 
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 2
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 2
-            </div>
-          </div>
-
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 3
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 3
-            </div>
-          </div>
-
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 4
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 4
-            </div>
-          </div>
+        <div className="bg-base-200/20 backdrop-blur-xs">
+          <div className="w-[90%] landscape:w-full h-0.5 ml-[5%] landscape:ml-0 bg-cyber_teal"></div>
         </div>
       </div>
     </>
diff --git a/frontend/src/components/ui_components/home_route/footer/Footer.jsx b/frontend/src/components/ui_components/home_route/footer/Footer.jsx
index ed5445a..dc7e3c0 100755
--- a/frontend/src/components/ui_components/home_route/footer/Footer.jsx
+++ b/frontend/src/components/ui_components/home_route/footer/Footer.jsx
@@ -4,7 +4,10 @@ import igLogo from "../../../../assets/logos/socials/Instagram_Glyph_Gradient.pn
 
 const Footer = () => {
   return (
-    <footer className="footer flex-col landscape:flex-row bg-base-300 text-neutral-content p-8 landscape:p-4 flex items-center justify-around">
+    <footer
+      id="footer"
+      className="footer flex-col landscape:flex-row bg-base-300 text-neutral-content p-8 landscape:p-4 flex items-center justify-around"
+    >
       <aside>
         <Logo className="h-[100px] w-auto" />
       </aside>
diff --git a/frontend/src/components/ui_components/home_route/navbar/NavBar.jsx b/frontend/src/components/ui_components/home_route/navbar/NavBar.jsx
index 4d91524..3df58ce 100755
--- a/frontend/src/components/ui_components/home_route/navbar/NavBar.jsx
+++ b/frontend/src/components/ui_components/home_route/navbar/NavBar.jsx
@@ -1,4 +1,3 @@
-import { Link } from "react-scroll";
 import ThemeSwitch from "../../ThemeSwitch";
 import useHasScrolled from "../../../../hooks/useHasScrolled";
 
@@ -8,7 +7,7 @@ const NavBar = ({ content }) => {
   return (
     <div id="top" className="drawer">
       <input id="my-drawer-3" type="checkbox" className="drawer-toggle z-30" />
-      <div className="drawer-content flex flex-col">
+      <div className="drawer-content flex flex-col scroll-smooth">
         {/* Navbar */}
         <div
           className={`navbar bg-base-300 w-full fixed top-0 z-10 ${hasScrolled ? "shadow-md dark:shadow-none shadow-neutral-400 dark:border-1 border-neutral-800" : "border-0 shadow-none"}`}
@@ -35,86 +34,78 @@ const NavBar = ({ content }) => {
                 </svg>
               </label>
             </div>
-            <Link
-              to={hasScrolled ? "landing" : "footer"}
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
+            <a
+              href={hasScrolled ? "#landing" : "#footer"}
               className="btn btn-ghost text-xl"
             >
               Surveillance SA
-            </Link>
+            </a>
           </div>
           <div className="navbar-center">
             <div className="hidden flex-none lg:block">
               <ul className="menu menu-horizontal gap-2">
                 {/* Navbar menu content here */}
                 <li>
-                  <Link
-                    to="landing"
-                    spy={true}
-                    smooth={true}
-                    duration={500}
-                    offset={-64}
-                    activeClass="menu-active"
-                    className=""
+                  <a
+                    href="#landing"
+                    className="transition-all duration-300 ease-in-out"
                   >
                     Home
-                  </Link>
+                  </a>
                 </li>
                 <li>
-                  <Link
-                    to="about"
-                    spy={true}
-                    smooth={true}
-                    duration={500}
-                    offset={-64}
-                    activeClass="menu-active"
-                    className=""
+                  <a
+                    href="#about"
+                    className="transition-all duration-300 ease-in-out"
                   >
                     About
-                  </Link>
+                  </a>
                 </li>
                 <li>
-                  <Link
-                    to="services"
-                    spy={true}
-                    smooth={true}
-                    duration={500}
-                    offset={-64}
-                    activeClass="menu-active"
-                    className=""
-                  >
-                    Services
-                  </Link>
+                  <details>
+                    <summary>
+                      <a
+                        href="#services"
+                        className="transition-all duration-300 ease-in-out"
+                      >
+                        Services
+                      </a>
+                    </summary>
+                    <ul className="p-2 min-w-max">
+                      <li>
+                        <a
+                          href="#service_1"
+                          className="transition-all duration-300 ease-in-out"
+                        >
+                          Submenu 1
+                        </a>
+                      </li>
+                      <li>
+                        <a
+                          href="#service_2"
+                          className="transition-all duration-300 ease-in-out"
+                        >
+                          Submenu 2
+                        </a>
+                      </li>
+                    </ul>
+                  </details>
                 </li>
                 <li>
-                  <Link
-                    to="gallery"
-                    spy={true}
-                    smooth={true}
-                    duration={500}
-                    offset={-64}
-                    activeClass="menu-active"
-                    className=""
+                  <a
+                    href="#gallery"
+                    className="transition-all duration-300 ease-in-out"
                   >
                     Gallery
-                  </Link>
+                  </a>
                 </li>
                 <li>
-                  <Link
-                    to="contact"
-                    spy={true}
-                    smooth={true}
-                    duration={500}
-                    offset={-64}
-                    activeClass="menu-active"
-                    className=""
+                  <a
+                    href="#contact"
+                    className="transition-all duration-300 ease-in-out"
                   >
                     Contact Us
-                  </Link>
+                  </a>
                 </li>
               </ul>
             </div>
@@ -137,84 +128,75 @@ const NavBar = ({ content }) => {
         <ul className="menu bg-base-200 min-h-full w-80 p-4 gap-2 top-[64px]">
           {/* Navbar menu content here */}
           <li>
-            <Link
-              to="landing"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
-              className=""
+            <a
+              href="#landing"
+              className="transition-all duration-300 ease-in-out"
               onClick={() => {
                 document.getElementById("my-drawer-3").checked = false;
               }}
             >
               Home
-            </Link>
+            </a>
           </li>
           <li>
-            <Link
-              to="about"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
-              className=""
+            <a
+              href="#about"
+              className="transition-all duration-300 ease-in-out"
               onClick={() => {
                 document.getElementById("my-drawer-3").checked = false;
               }}
             >
               About
-            </Link>
+            </a>
           </li>
           <li>
-            <Link
-              to="services"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
-              className=""
+            <a
+              href="#services"
+              className="transition-all duration-300 ease-in-out"
               onClick={() => {
                 document.getElementById("my-drawer-3").checked = false;
               }}
             >
               Services
-            </Link>
+            </a>
+            <ul className="p-2">
+              <li>
+                <a
+                  href="#service_1"
+                  className="transition-all duration-300 ease-in-out"
+                  onClick={() => {
+                    document.getElementById("my-drawer-3").checked = false;
+                  }}
+                >
+                  Surveillance and System Health Monitoring
+                </a>
+              </li>
+              <li>
+                <a>Submenu 2</a>
+              </li>
+            </ul>
           </li>
           <li>
-            <Link
-              to="gallery"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
-              className=""
+            <a
+              href="#gallery"
+              className="transition-all duration-300 ease-in-out"
               onClick={() => {
                 document.getElementById("my-drawer-3").checked = false;
               }}
             >
               Gallery
-            </Link>
+            </a>
           </li>
           <li>
-            <Link
-              to="contact"
-              spy={true}
-              smooth={true}
-              duration={500}
-              offset={-64}
-              activeClass="menu-active"
-              className=""
+            <a
+              href="#contact"
+              className="transition-all duration-300 ease-in-out"
               onClick={() => {
                 document.getElementById("my-drawer-3").checked = false;
               }}
             >
               Contact Us
-            </Link>
+            </a>
           </li>
         </ul>
       </div>
diff --git a/frontend/src/components/ui_components/icons/CamerasOnPoleIcon.jsx b/frontend/src/components/ui_components/icons/CamerasOnPoleIcon.jsx
index 6f9bc01..2f76af4 100644
--- a/frontend/src/components/ui_components/icons/CamerasOnPoleIcon.jsx
+++ b/frontend/src/components/ui_components/icons/CamerasOnPoleIcon.jsx
@@ -1,16 +1,10 @@
-const CamerasOnPoleIcon = ({
-  className = "",
-  width = "6.9412537mm",
-  height = "9.5249987mm",
-}) => {
+const CamerasOnPoleIcon = ({ className = "" }) => {
   const pathClass = "fill-[#009A99] fill-opacity-100 stroke-0";
 
   return (
     <>
       <svg
-        className={className}
-        width={width}
-        height={height}
+        className={`w-[6.9412537mm] h-[9.5249987mm] ${className}`}
         viewBox="0 0 6.9412537 9.5249987"
         version="1.1"
         id="svg1"
diff --git a/frontend/src/index.css b/frontend/src/index.css
index a14bc2d..dc91829 100644
--- a/frontend/src/index.css
+++ b/frontend/src/index.css
@@ -92,3 +92,7 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
+
+[id] {
+  scroll-margin-top: 64px;
+}
diff --git a/frontend/temp/Services_1.jsx b/frontend/temp/Services_1.jsx
deleted file mode 100644
index 6353b9c..0000000
--- a/frontend/temp/Services_1.jsx
+++ /dev/null
@@ -1,147 +0,0 @@
-import CamerasOnPoleIcon from "../../../ui_components/icons/CamerasOnPoleIcon";
-
-const Services = () => {
-  return (
-    <>
-      {/* name of each tab group should be unique */}
-      <div className="w-[calc(100%_-_2rem)] landscape:w-[80vw] h-[calc(100dvh_-_150px)] flex items-center justify-center">
-        <div className="tabs tabs-lift w-full">
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" defaultChecked />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 1
-          </label>
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-contain bg-repeat-round bg-left-top bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)] flex flex-col items-center gap-8">
-              <div className="flex flex-col justify-center items-center gap-4">
-                <h2 className="text-cyber_teal font-bold text-3xl text-center">
-                  Surveillance and System Health Monitoring
-                </h2>
-                <p className="text-center text-sm max-w-[70%]">
-                  We provide uninterrupted protection and peace of mind.
-                  Ensuring that even if the camera is stolen, damaged or loses
-                  connection, your footage is always safe and your system is
-                  always under watch.
-                </p>
-              </div>
-              <div className="w-full flex flex-col gap-4">
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    REDUNDANT FOOTAGE BACKUP
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    LIVE SYSTEM HEALTH MONITORING
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">
-                    LESS DOWNTIME GUARANTEE
-                  </h3>
-                </div>
-                <div className="flex w-full flex-row justify-start items-center gap-4">
-                  <CamerasOnPoleIcon className="" width="48" height="48" />
-                  <h3 className="text-xl font-semibold">TRUE PEACE OF MIND</h3>
-                </div>
-              </div>
-            </div>
-          </div>
-
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 2
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 2
-            </div>
-          </div>
-
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 3
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 3
-            </div>
-          </div>
-
-          <label className="tab has-[input:checked]:text-cyber_teal">
-            <input className="" type="radio" name="my_tabs_4" />
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              fill="none"
-              viewBox="0 0 24 24"
-              strokeWidth="1.5"
-              stroke="currentColor"
-              className="size-4 me-2"
-            >
-              <path
-                strokeLinecap="round"
-                strokeLinejoin="round"
-                d="M5.25 5.653c0-.856.917-1.398 1.667-.986l11.54 6.347a1.125 1.125 0 0 1 0 1.972l-11.54 6.347a1.125 1.125 0 0 1-1.667-.986V5.653Z"
-              />
-            </svg>
-            Tab 4
-          </label>
-
-          <div className="tab-content h-[calc(100dvh_-_150px_-_40px)] bg-base-100 border-base-300 p-6 landscape:bg-[url('/services/desktop/service1ll.png')] dark:landscape:bg-[url('/services/desktop/service1ld.png')] bg-cover bg-center bg-no-repeat bg-clip-border">
-            <div className="w-full h-[calc(100dvh_-_150px_-_40px)]  flex justify-center items-center">
-              Tab content 4
-            </div>
-          </div>
-        </div>
-      </div>
-    </>
-  );
-};
-
-export default Services;
diff --git a/frontend/temp/icons/jsx/CertifiedInstallerIcon.jsx b/frontend/temp/icons/jsx/CertifiedInstallerIcon.jsx
index b11f936..ae3deac 100644
--- a/frontend/temp/icons/jsx/CertifiedInstallerIcon.jsx
+++ b/frontend/temp/icons/jsx/CertifiedInstallerIcon.jsx
@@ -1,17 +1,11 @@
-const CertifiedInstallerIcon = ({
-  className = "",
-  width = "36",
-  height = "36",
-}) => {
+const CertifiedInstallerIcon = ({ className = "" }) => {
   const pathClass =
     "fill-base-content dark:fill-[#009A99] fill-opacity-100 inline";
 
   return (
     <>
       <svg
-        className={className}
-        width={width}
-        height={height}
+        className={`w-[36px] h-[36px] ${className}`}
         viewBox="0 0 9.5249995 9.5250003"
         version="1.1"
         id="svg1"
diff --git a/frontend/temp/icons/jsx/HumanAndVehicleAiIcon.jsx b/frontend/temp/icons/jsx/HumanAndVehicleAiIcon.jsx
index 990b8ac..6802b74 100644
--- a/frontend/temp/icons/jsx/HumanAndVehicleAiIcon.jsx
+++ b/frontend/temp/icons/jsx/HumanAndVehicleAiIcon.jsx
@@ -1,16 +1,10 @@
-const HumanAndVehicleAiIcon = ({
-  className = "",
-  width = "36",
-  height = "36",
-}) => {
+const HumanAndVehicleAiIcon = ({ className = "" }) => {
   const pathClass = "fill-base-content dark:fill-[#009A99] fill-opacity-100";
 
   return (
     <>
       <svg
-        className={className}
-        width={width}
-        height={height}
+        className={`w-[36px] h-[36px] ${className}`}
         viewBox="0 0 9.5249995 9.5250003"
         version="1.1"
         id="svg1"
diff --git a/frontend/temp/icons/jsx/LiveMonitoringIcon.jsx b/frontend/temp/icons/jsx/LiveMonitoringIcon.jsx
index 83c848b..8fcbaef 100644
--- a/frontend/temp/icons/jsx/LiveMonitoringIcon.jsx
+++ b/frontend/temp/icons/jsx/LiveMonitoringIcon.jsx
@@ -1,17 +1,11 @@
-const LiveMonitoringIcon = ({
-  className = "",
-  width = "36",
-  height = "36",
-}) => {
+const LiveMonitoringIcon = ({ className = "" }) => {
   const pathClass =
     "fill-base-content dark:fill-[#009A99] fill-opacity-100 stroke-[0.352778]";
 
   return (
     <>
       <svg
-        className={className}
-        width={width}
-        height={height}
+        className={`w-[36px] h-[36px] ${className}`}
         viewBox="0 0 9.5249995 9.5250003"
         version="1.1"
         id="svg1"
diff --git a/frontend/temp/icons/jsx/SolarAndRemoteIcon.jsx b/frontend/temp/icons/jsx/SolarAndRemoteIcon.jsx
index aa030aa..7d77266 100644
--- a/frontend/temp/icons/jsx/SolarAndRemoteIcon.jsx
+++ b/frontend/temp/icons/jsx/SolarAndRemoteIcon.jsx
@@ -1,17 +1,11 @@
-const SolarAndRemoteIcon = ({
-  className = "",
-  width = "36",
-  height = "36",
-}) => {
+const SolarAndRemoteIcon = ({ className = "" }) => {
   const pathClass =
     "fill-base-content dark:fill-[#009A99] fill-opacity-100 stroke-[0.249103]";
 
   return (
     <>
       <svg
-        className={className}
-        width={width}
-        height={height}
+        className={`w-[36px] h-[36px] ${className}`}
         viewBox="0 0 9.5249995 9.5250003"
         version="1.1"
         id="svg1"
diff --git a/todo.md b/todo.md
index 01ccc38..626646b 100644
--- a/todo.md
+++ b/todo.md
@@ -1,5 +1,6 @@
 # TO-DO
 
+- [ ] add useEffect to NavBar.jsx to add 'menu-active' class based on scroll position.
 - [ ] Fix preloading of globe assets
 - [ ] Refactor gallery images to use a loop
 - [ ] Complete rewrite of landing
